app/src/main/res/values-hi/strings.xml

```diff
@@ -0,0 +1,127 @@
+<string name="access_denied">पहुंच अस्वीकृत</string>
```

Comment:
Translation looks good but some translation aren't quite proper

Reply:
Fuck chatgpt

Reply:
You changed bundle.gradle?

Reply:
what is that?